resources/model/OpenStudio.idd

```diff
@@ -35689,6 +35689,25 @@ OS:OutputControl:Files,
        \key No
        \required-field
 
+OS:OutputControl:Timestamp,
+       \memo Control timestamp format, currently applies only to JSON and native CSV (not CSV via ReadVars)
+       \unique-object
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2,  \field ISO 8601 Format
+       \note Use the ISO 8601 format YYYY-MM-DDThh:mm:ss
+       \type choice
+       \key Yes
+       \key No
+       \required-field
+  A3;  \field Timestamp at Beginning of Interval
```

Comment:
```suggestion
  A3;  \field Timestamp At Beginning of Interval
```

if we want everything to match.

Reply:
Even though, e.g., SizingPeriod:DesignDay isn't like this?

Reply:
The method name is what I care about.

But if you do shenanigans[*] like getting an IDD field to infer what the getter/setter corresponding to it is, changing the case in the IDD to match is better.

[*] These "shenanigans" are **exactly** what GenerateClass.rb does

Reply:
I haven't checked thoroughly, but I think we tend to prefer using "Of" too instead of "of".